### PR TITLE
#118: Updated min python version 3.8

### LINKED ISSUE #118: Update minimum Python version to 3.8
## Background

Currently the minimum version is set to Python 3.6. However, Python 3.6 is already EOL, and many packages stopped support for it.
- Requires 
  - https://github.com/exasol/exasol-python-test-framework/issues/37
  - https://github.com/exasol/integration-test-docker-environment/issues/192

## Acceptance Criteria
- Update minimum Python version to 3.8 in Poetry
- Update Docker-Start-Container to Ubuntu 20.04 and Python3.8 (https://github.com/exasol/script-languages-container-tool/blob/main/docker_runner/Dockerfile)
- Update Docker-Test-Container to Ubuntu 20.04 and Python 3.8 (https://github.com/exasol/script-languages-container-tool/blob/main/tests/Dockerfile)

### PR DESCRIPTION
**Pull request by MarleneKress79789**
fixes #118

### CODE REVIEW

File: docker_runner/Dockerfile [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 COPY ext/01_nodoc /etc/dpkg/dpkg.cfg.d/01_nodoc
 
 RUN apt-get -y update && \
     apt-get -y install --no-install-recommends\
         ca-certificates \
         locales \
-        python3-wheel \
-        python3-setuptools \
+        python3.8-venv \
```
Comment (tkilias) [inline docker_runner/Dockerfile:10]: why did you change this to python3.8-venv, does not python3.8-wheel or setuptools exists?

**Review by tkilias — CHANGES_REQUESTED** [review]

File: docker_runner/Dockerfile (hunk 1 above)
Comment (tkilias) [reply docker_runner/Dockerfile:10]: I had also a look, it seems so. The only thing you could do, install its dependencies individually, but this is overkill.
https://packages.ubuntu.com/focal/python3.8-venv

**Review by tomuben — APPROVED** [review]
